README.md

```diff
@@ -1,7 +1,6 @@
 Faraday Middleware
 ==================
 [![Gem Version](https://badge.fury.io/rb/faraday_middleware.svg)](https://rubygems.org/gems/faraday_middleware)
-[![Build Status](https://travis-ci.org/lostisland/faraday_middleware.svg)](https://travis-ci.org/lostisland/faraday_middleware)
```

Comment:
Would it be possible to have the GitHub Actions CI badge like on Faraday 😄?

Reply:
Probably: I'll add it to the list of things!

Reply:
There, added a SVG badge.